include/config/config_game.h

```diff
@@ -1,4 +1,6 @@
 #ifndef CONFIG_GAME_H
 #define CONFIG_GAME_H
 
+#include "config_graphics.h"
```

Comment:
is there a base config file? I would expect include/config.h to include all of these sub config files